src/main/java/org/jenkinsci/test/acceptance/po/Control.java

```diff
@@ -103,7 +105,14 @@ public void check(boolean state) {
      * @see #clickAndWaitToBecomeStale(Duration)
      */
     public void click() {
-        resolve().click();
+        WebElement we = resolve();
+        // button may be obscured by say the "Save Apply" screen so we wait as Selenium will do a scroll but the CSS 
+        // can take a while to update the layout \o/
+        waitFor(we).
```

Comment:
@aHenryJard this is the flake in the MCP tests.

I would prefer a method to actually check if something is clickable rather than trying repeatedly but I could not find anything.